### ISSUE TITLE
Swipe then touch on table view is not triggering transition

### ISSUE THREAD

**Issue by newtewt** [calabash/calabash-ios#1312]
I am having an issue with swiping then touching a cell in a tableview. Can reproduce the issue using the console. Reproduction steps below. Using the latest versions of gem and server. Running xcode 8.3.2 and iOS 10.3.2 on a physical device. 

To reproduce you need a tableview with enough cells that cause a scroll bar and when touching the cell it moves to another page.

In calabash console:
Run `start_test_server_in_background`
Navigate to the tableview in the app
Run `swipe :up`
Run `touch "UITableViewCell index:3"`  Note: 3 is arbitrary number to try and indicate a cell index that is completely visible.
The cell shows the state change but doesn't go forward.
Run `touch "UITableViewCell index:3"`  
The app goes to the next page.

**Comment by vitaplena** [comment]
Some suggestions that come to mind:
1. `wait_for_none_animating` before touching anything.
2. `wait_tap` on the control instead of `touch` to guarantee it is found in the screen before the action
3. Make sure nothing is on top of the control you are trying to tap
4. Maybe tap on another control within the cell

Good luck!

**Comment by newtewt** [comment]
@vitaplena  I don't believe it is a timing issue because I am manually recreating the issue while running in console. So I wait until everything has stopped moving and ensure the element is visible. 
There is nothing on top of the cell that I can see.
I've tried the label inside and the cell.

**Comment by newtewt** [comment]
I've created a very basic iOS app and a very basic test suite that can be used to recreate the issue. 

iOS app: https://github.com/newtewt/calabash_ios_bug_example
Ruby Test Suite: https://github.com/newtewt/ruby_test_suite_for_bug

**Comment by jmoody** [comment]
@newtewt Thanks for the sample project.  Are you still having problems?

**Comment by newtewt** [comment]
@jmoody  I no longer work for the company using Calabash. When I left in august I believe it was still an issue.
